src/api/app/models/notification.rb

```diff
@@ -2,6 +2,8 @@ class Notification < ApplicationRecord
   MAX_RSS_ITEMS_PER_USER = 10
   MAX_RSS_ITEMS_PER_GROUP = 10
 
+  self.ignored_columns = ['type']
```

Comment:
This line of code, shouldn't be here, right? If I'm not wrong we removed it in a previous PR.

Reply:
Dropped!